tests/InputFiles/GeneralizedHarmonic/BinaryBlackHole.yaml

```diff
@@ -19,6 +19,15 @@ ResourceInfo:
     ControlSystemAhB:
       Proc: Auto
       Exclusive: true
+    MemoryMonitor:
+      Proc: Auto
+      Exclusive: false
+    Rotation:
+      Proc: Auto
+      Exclusive: false
+    Expansion:
+      Proc: Auto
+      Exclusive: false
```

Comment:
Not a change request for you, but for me. After this PR is merged, I should make the `SingletonPack` use an `Options::Auto<SingletonInfoHolder>` instead of just a `SingletonInfoHolder` so these lines can just be

```yaml
    MemoryMonitor: Auto
    Rotation: Auto
    Expansion: Auto
```

which looks a lot cleaner imo.

Reply:
Oh, yes I like this